.github/workflows/conda-forge-ci.yml

```diff
@@ -0,0 +1,73 @@
+name: C++ CI Workflow with conda dependencies
+
+on:  
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Execute a "nightly" build at 2 AM UTC
+  - cron:  '0 2 * * *'
+
+jobs:
+  build:
+    name: '[${{ matrix.os }}@${{ matrix.build_type }}@conda]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build_type: [Release]
+        os: [ubuntu-latest, windows-2019]
```

Comment:
Why `windows-2019` and not `windows-latest`?

Reply:
See https://github.com/robotology/yarp-device-xsensmt/pull/31/files#r868948384 .